tensorflow_datasets/scripts/replace_fake_images.py

```diff
@@ -116,8 +115,8 @@ def rewrite_tar(root_dir, tar_filepath):
     rewrite_dir(temp_dir)  # Recursivelly compress the archive content
 
     # Converting into tarfile again to decrease the space taken by the file-
-    with tarfile.open(tar_filepath, 'w' + extension) as tar:
-      tar.add(temp_dir, recursive=True)
+    with tarfile.open(tar_filepath, 'w:gz') as tar:
+      tar.add(temp_dir, recursive=True,arcname=os.path.basename(temp_dir))
```

Comment:
Can you add a comment explaining why `arcname` is required here in this context, as reader might not be familiar with it. (e.g. give an example of filepath)

Reply:
Yeah sure its good for reader also, as I explains above why I use `arcname `here